packages/expo/src/Notifications/Notifications.ts

```diff
@@ -438,4 +441,52 @@ export default {
     }
     return ExponentNotifications.setBadgeNumberAsync(number);
   },
+
+  async scheduleNotificationWithCalendarAsync(
+    notification: LocalNotification,
+    options: {
+      year?: number;
+      month?: number;
+      hour?: number;
+      day?: number;
+      minute?: number;
+      second?: number;
+      weekDay?: number;
+      repeat?: boolean;
+    } = {}
+  ): Promise<string> {
+    const areOptionsValid: boolean =
+      (options.month == null || isInRange(options.month,1,12)) &&
+      (options.day == null || isInRange(options.day, 1, 31)) &&
+      (options.hour == null || isInRange(options.hour, 0, 23)) &&
+      (options.minute == null || isInRange(options.minute, 0, 59)) &&
+      (options.second == null || isInRange(options.second, 0, 59)) &&
+      (options.weekDay == null || isInRange(options.weekDay, 1, 7)) &&
+      (options.weekDay == null || options.day == null);
+
+    if (!areOptionsValid) {
+      console.warn('Options in scheduleNotificationWithCalendarAsync call were incorrect!');
+      return "unsuccessful";
+    }
+
+    return ExponentNotifications.scheduleNotificationWithCalendar(notification, options);
+  },
+
+  async scheduleNotificationWithTimerAsync(
+    notification: LocalNotification,
+    options: {
+      interval: number;
+      repeat?: boolean;
+    } 
+  ): Promise<string> {
+    if (options.interval < 1) {
+      console.warn('Interval must be not less then 1');
+      return "unsuccessful";
```

Comment:
I think we could throw an error here, otherwise someone could write a logic that could think that `unsuccessful` is actually a valid notification ID.